### ISSUE TITLE
Thank you!

### ISSUE THREAD

**Issue by foxx** [hay/stapes#51]
No problem here, just wanted to say thank you giving us this library, it's really given me everything I need to create clean, beautiful JS code. Keep up the great work!

**Comment by RWOverdijk** [comment]
@supermensa It doesn't have to be 1.0.0, does it? That would mean a set of breaking changes. I don't think that [1.0.0 means what you think it means](http://semver.org/).

@hay I'd like to toss in my compliment, too. It's a shame it doesn't help out with sorting number-indexed arrays, but I can live with that for now. Bottom line: nice job.

**Comment by supermensa** [comment]
@RWOverdijk 
Yeah, it depends on [who you ask](https://github.com/jashkenas/underscore/issues/1805).
What I meant by a version 1.0.0: When will Stapes be considered ready for production?

**Comment by RWOverdijk** [comment]
@supermensa That's true. Let's just say that the vast majority (as far as I can see, I'm no authority on this, obviously) uses semver.

I figured. I'd say now :) It's being used in production already (by me as well) and does what it says it does. It's not that big either so really, not a lot that can go wrong.

**Comment by zgoda** [comment]
Thank you, this is really great small piece of software. Mind you, I have 3 MB storage on my "server" (ESP8266-based microcontroller) so your library is a gem for me. :1st_place_medal:
